driver-core/src/main/com/mongodb/internal/capi/MongoCryptHelper.java

```diff
@@ -72,9 +78,31 @@ public static MongoCryptOptions createMongoCryptOptions(final Map<String, Map<St
             spawnArgs.add("--idleShutdownTimeoutSecs");
             spawnArgs.add("60");
         }
+        if (!spawnArgs.contains("--logpath")) {
+            spawnArgs.add("--logpath");
+            spawnArgs.add(System.getProperty("os.name").startsWith("Windows") ? "NUL" : "/dev/null");
+        }
         return spawnArgs;
     }
 
-    private MongoCryptOptionsHelper(){
```

Comment:
You need to add a private MongoCryptHelper constructor.

Reply:
Done